src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCONT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCONT = 20;
+  const BASE_COST = 40;
+
+  const basePrice = days * BASE_COST;
```

Comment:
Please use one rule for writing all constants names. All constants names must be capital letters